spec/models/user_spec.rb

```diff
@@ -23,6 +23,12 @@
     expect(@user).not_to be_valid
   end
 
+  it "(student) is not valid without a calender url" do
```

Comment:
Why are there brackets?

Reply:
I dunno, that's Lucs code, but I guess we could just leave them out.